mteb/tasks/Classification/eng/LegalBenchClassification.py

```diff
@@ -1500,8 +1500,440 @@ class CUADGoverningLawLegalBenchClassification(AbsTaskClassification):
+        form=["written"],
+        domains=["Legal"],
+        task_subtypes=[],
+        license="cc-by-4.0",
+        socioeconomic_status="high",
+        annotations_creators="expert-annotated",
+        dialect=[],
+        text_creation="found",
+        bibtex_citation="""
+        @misc{guha2023legalbench,
+            title={LegalBench: A Collaboratively Built Benchmark for Measuring Legal Reasoning in Large Language Models}, 
+            author={Neel Guha and Julian Nyarko and Daniel E. Ho and Christopher Ré and Adam Chilton and Aditya Narayana and Alex Chohlas-Wood and Austin Peters and Brandon Waldon and Daniel N. Rockmore and Diego Zambrano and Dmitry Talisman and Enam Hoque and Faiz Surani and Frank Fagan and Galit Sarfaty and Gregory M. Dickinson and Haggai Porat and Jason Hegland and Jessica Wu and Joe Nudell and Joel Niklaus and John Nay and Jonathan H. Choi and Kevin Tobia and Margaret Hagan and Megan Ma and Michael Livermore and Nikon Rasumov-Rahe and Nils Holzenberger and Noam Kolt and Peter Henderson and Sean Rehaag and Sharad Goel and Shang Gao and Spencer Williams and Sunny Gandhi and Tom Zur and Varun Iyer and Zehua Li},
+            year={2023},
+            eprint={2308.11462},
+            archivePrefix={arXiv},
+            primaryClass={cs.CL}
+        },
+        @article{hendrycks2021cuad,
+            title={Cuad: An expert-annotated nlp dataset for legal contract review},
+            author={Hendrycks, Dan and Burns, Collin and Chen, Anya and Ball, Spencer},
+            journal={arXiv preprint arXiv:2103.06268},
+            year={2021}
+        }
+        """,
+        n_samples={"test": 1030},
+        avg_character_length={"test": 365.54},
+    )
+
+    def dataset_transform(self):
+        mapping = {"yes": 1, "no": 0}
+        self.dataset = self.dataset.map(
+            lambda example: {
+                "answer": mapping.get(example["answer"].lower(), example["answer"])
+            }
+        )
+        self.dataset = self.dataset.rename_column("answer", "label")
+
+
+class CUADIPOwnershipAssignmentLegalBenchClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="CUADIPOwnershipAssignmentLegalBenchClassification",
+        description="This task was constructed from the CUAD dataset. It consists of determining if the clause specifies that intellectual property created by one party become the property of the counterparty, either per the terms of the contract or upon the occurrence of certain events.",
+        reference="https://huggingface.co/datasets/nguha/legalbench",
+        dataset={
+            "path": "nguha/legalbench",
+            "name": "cuad_ip_ownership_assignment",
+            "revision": "12ca3b695563788fead87a982ad1a068284413f4",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="accuracy",
+        date=("2023-08-23", "2023-08-23"),
```

Comment:
This does not seem to reflect the time period in which the text was written

Reply:
I understand. I was not aware that we reported the date of creation of the texts as well, hence I was only mentioning the final date of curation. 

The date for all the `CUAD` datasets has been updated to `"2021-03-11"` based on their [release date](https://zenodo.org/records/4595826). I also have updated the date for the `ContractNLI` datasets to `"2021-11-01"` based on the date the [original paper](https://aclanthology.org/2021.findings-emnlp.164/) was published. The final date has been left at `"2023-08-23"`, since the LegalBench authors have curated and modified the original datasets.

Reply:
Sounds like to text creation (when it was written) is probably before "2021-03-11". I would probably make it 2000-01-01, if you don't have a better guess. I could imagine that some of the legal documents are even older though.

Reply:
The `ContractNLI` paper mentions the documents are from 1996 to 2020. The `CUAD` paper does not mention the exact dates, but the scraping code they released shows the files are from 2000-2020. Should the final date be `"2020-12-31"` (and not `"2023-08-23"`, when the LegalBench paper was released)?

Reply:
I would set the final date to the last time you believe the text was modified (2023).

Reply:
Got it! I have updated the dates in f7867894875358d9752be3b36005c190a92ca1e4. Thanks for clarifying!